shared/wallets/wallet/header/wallet-switcher/index.native.js

```diff
@@ -148,7 +154,7 @@ export const WalletSwitcher = (props: Props) => {
 const styles = Styles.styleSheetCreate({
   container: {
     justifyContent: 'flex-end',
-    maxHeight: '50%',
+    maxHeight: '100%',
```

Comment:
design has 100%, and @cecileboucheron confirmed that's what's desired, so just using that